### ISSUE TITLE
Release version of bitters depends on a beta version of Bourbon?

### ISSUE THREAD

**Issue by Ziao** [thoughtbot/bitters#239]
Unless I am mistaken, does the latest Bitters really depend on v5 of Bourbon, which is still in beta? If so, that's quite uncommon. What's the idea behind this?

**Comment by tysongach** [comment]
Duplicate of https://github.com/thoughtbot/bitters/issues/238.
